packages/dds/tree/src/test/simple-tree/api/schemaFactoryRecursive.spec.ts

```diff
@@ -715,4 +756,130 @@ describe("SchemaFactory Recursive methods", () => {
 		const r = hydrate(Root, { r: new ArrayRecursive([]) });
 		assert.deepEqual([...r.r], []);
 	});
+
+	/**
+	 * These cases are anti-patterns which are documented not to work, but still might get used my someone.
+	 * SchemaFactory documents the non explicit sub-classing pattern (POJO mode) is not supported for recursive schema.
+	 * Additionally it documents this pattern is not supported for Explicitly named Maps and Arrays.
+	 *
+	 * These tests violate both of these statements, seeing which version currently compile and which ones do not.
+	 * Having these tests in place will help know when.if be further break cases like these to adi in writing helpful changesets for impacted customers.
+	 *
+	 * These patterns also [break type safety in .d.ts generation](https://github.com/microsoft/TypeScript/issues/55832):
+	 * this is one of the reasons they are not supported.
+	 * The import-testing has test coverage for aspect.
+	 * They also have poorer error quality and IntelliSense, and tend to fail to compile in some cases.
+	 *
+	 * These tests are all about the Typing.
```

Comment:
Is this meant to be capitalized?

```suggestion
	 * These tests are all about the typing.
```